granulate_utils/linux/cgroups/memory_cgroup.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+from granulate_utils.linux.cgroups.base_cgroup import BaseCgroup
+from granulate_utils.linux.cgroups.exceptions import MissingController
+
+
+class MemoryCgroup(BaseCgroup):
+    HIERARCHY = "memory"
+
+    def get_memory_limit(self) -> int:
+        return int(self.read_from_controller("memory.limit_in_bytes"))
+
+    def get_max_memory(self) -> int:
```

Comment:
Let's use the exact names as cgroup uses to avoid confusion. So
```suggestion
    def get_max_usage_in_bytes(self) -> int:
```